blocks/block-alignment-toolbar/index.js

```diff
@@ -53,3 +58,9 @@ export default function BlockAlignmentToolbar( { value, onChange, controls = DEF
 		/>
 	);
 }
+
+export default withEditorSettings(
+	( settings ) => ( {
```

Comment:
I'm planning to introduce the blockTypes, fallbackBlockName.... to these settings and I expect several components to extract a single or multiple blockTypes from the settings. Mapping helps use factorize this using functions similar to selectors to avoid duplication.

Also, It's nice that the component prop is `wideAlignmentsEnabled` instead of leaving the responsibility of extracting the prop to the component itself.